jolocom-did-driver/index.ts

```diff
@@ -0,0 +1,22 @@
+import express from "express"
+import { getResolver } from "jolo-did-resolver";
+import { Resolver } from "did-resolver";
+
+const PORT = 8080;
+
+const app = express();
+app.get('/1.0/identifiers/:did', async (req, res) => {
+  const did = req.params.did;
+  console.log(did);
```

Comment:
Do we need this?

Reply:
just added that because there where other console.logs before, wasn't sure if this is maybe used for logging